Adds traffic page (#3888)
wso2/docs-apim: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md

### Modern Responsive UIs
With cloud-based analytics now we have sleek and modern responsive UIs with improved 
 filtering and drill down capabilities. 

> You can access the all new analytics portal through [here](https://analytics.choreo.dev/). Use 

Reviewer: ```suggestion
You can access the new analytics portal from [here](https://analytics.choreo.dev/).
```

> [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction) document as a reference to the

Reviewer: ```suggestion
To get an overview of the available dashboards and its capabilities, see the [introduction]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction).
```

>  composition of each dashboard and information that is available through them. 

Reviewer: ```suggestion
```
